app/controllers/media_controller.rb

```diff
@@ -97,6 +97,10 @@ def proxy_to_s3_via_nginx(asset)
 
     if request.fresh?(response)
       head :not_modified
+    elsif AssetManager.s3.fake?
+      url = Services.cloud_storage.presigned_url_for(asset, http_method: request.request_method)
+      redirect_to url
+      nil
```

Comment:
what's this nil return value for? I'm assuming this would never get hit if it's redirecting?

Reply:
Thanks for pointing that out. After reviewing the other usages for redirect url, we noticed that it wasn't necessary to return nil. We have now removed it.

Reply:
Do you mind pushing that change up? The `nil` is still there

Reply:
The change has been pushed now, thank you!!